app/views/static/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= javascript_pack_tag 'application' %>
```

Comment:
- [ ] The view for the root route should be created (root#index) and includes only `<div id="root">`, the rest should be done with React and Redux, kindly make sure to implement this

Reply:
I have overwriten this file app/views/static/index.html.erb with "<div id="root">"
Thank you so much :+1: